server/schema/donations.sql

```diff
@@ -5,7 +5,7 @@ CREATE TYPE status AS ENUM ('pending', 'approved', 'denied', 'flagged', 'schedul
 
 CREATE TABLE donations (
    id SERIAL PRIMARY KEY,
-   route_id INT,
+   route_id INT REFERENCES routes(id) ON DELETE CASCADE,
```

Comment:
If a route gets deleted, all donations associated with that route should not be deleted. The exact action is still to be determined, so for now, change the `on delete cascade` to `no action`.

Reply:
This change was made to the database tables already. Code may not have been pushed.